package.json

```diff
@@ -92,7 +92,10 @@
     "testPathIgnorePatterns": [
       ".d.ts"
     ],
-    "testTimeout": 5000
+    "testTimeout": 5000,
+    "moduleNameMapper": {
+      "^axios$": "axios/dist/node/axios.cjs"
```

Comment:
what is this for?

Reply:
not familiar with js but this is the reason https://stackoverflow.com/questions/73958968/cannot-use-import-statement-outside-a-module-with-axios

Reply:
I had the same issue when updating Faros CE and resolved it bumping the jest unit-testing library and related testing libraries (ts-jest, @types/jest), to the [latest versions](https://github.com/faros-ai/faros-community-edition/pull/323/files)

Reply:
ok thanks the info. let me try to do the same

Reply:
@cjwooo i see you're using `1.6.0`. does it matter which `1.6.x` we upgrade?

Reply:
go with the latest